presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -1,55 +1,63 @@
-Currently the only connector supporting the mechanism is Hive connector.
+Table statistics are provided to the query planner by connectors.
+Currently the only connector that supports statistics is the Hive connector.
 
-Table layouts
+Table Layouts
 -------------
 
-Statistics are exposed to query planner on per table layout basis. Table layout represents a subset of table's data with attached
-data organization properties (like sort order, bucketing etc.).
+Statistics are exposed to the query planner by table layout. A table layout represents a subset of a table's data
+and contains information about the organizational properties of that data (like sort order and bucketing).
 
-How many table layouts table has and details of table layout implementation are specific to connector.
-Taking Hive connector as an example:
+The number of table layouts available for a table and the details of those table layouts are specific to each connector.
+Using the Hive connector as an example:
 
-* not partitioned tables have just one table layout - representing all data in a table
-* partitioned tables have a family of table layouts. Each subset of partitions to be scanned represent one table layout.
-  Presto will try to pick up smallest possible layout based on filtering predicates from query.
+* Non-partitioned tables have just one table layout representing all data in the table
+* Partitioned tables have a family of table layouts. Each set of partitions to be scanned represents one table layout.
+  Presto will try to pick a table layout consisting of the smallest number of partitions based on filtering predicates
+  from the query.
 
-Available statistics
+Available Statistics
 --------------------
 
-Currently following statics are available in Presto:
- * **rows count** - total number of rows for table layout
- * Per each column:
+Currently, the following statistics are available in Presto:
 
-   * **data size** - summary column data size which needs to be read
-   * **nulls count** - number of null values
-   * **distinct values count** - estimated number of distinct values
+ * For the whole table:
 
+   * **row count**: the total number of rows for the table layout
 
-Depending which connector is in use only, subset of statistics may be available.
-For example Hive connector does not support data size for now.
-Also some statistics may be available for one table but not the other,
-or even one table layout and not the other for single table.
+ * For each column:
+
+   * **data size**: the data size that needs to be read
+   * **null count**: the number of null values
+   * **distinct value count**: the estimated number of distinct values
 
-Displaying table statistics
----------------------------
 
-Table statistics can be displayed via Presto CLI using ``SHOW STATS`` command.
+The actual statistics available for a particular query depends on the connector being used and can also vary by table or
```

Comment:
hmmm... depend sounds weird to me but I don't know why.  I think implicitly what we're talking about is the _set_ of available statistics, which is singular.  Maybe explicitly say "The set of available statistics..." so that  the rest of the sentence is definitely right.